From 39085f7f14b550e9f270ebddee736f9c7b5a36c9 Mon Sep 17 00:00:00 2001
From: Stefan Kost <ensonic@users.sf.net>
Date: Thu, 23 Jul 2009 11:32:02 +0300
Subject: [PATCH] camerabin: disable audioconversion elements (maemo specific)

We don't need those als pulse can supply whatever we need.
---
 gst/camerabin/camerabinvideo.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/gst/camerabin/camerabinvideo.c b/gst/camerabin/camerabinvideo.c
index bc91524..9a49902 100644
--- a/gst/camerabin/camerabinvideo.c
+++ b/gst/camerabin/camerabinvideo.c
@@ -69,7 +69,7 @@
 #define DEFAULT_MUX "oggmux"
 #define DEFAULT_SINK "filesink"
 
-#define USE_AUDIO_CONVERSION 1
+//#define USE_AUDIO_CONVERSION 1
 
 enum
 {
